src/commons/Post.tsx

```diff
@@ -0,0 +1,347 @@
+import React, { FunctionComponent, useCallback, useState } from 'react';

```

Comment:
When I open a post the app crashes with this error "Text strings must be rendered within a Text component

Reply:
I think it's because of line 303 on HtmlRenderer
You need to add !!htmlWidth